docs/content/Schema/Getting-Started-with-YAML.mdx

```diff
@@ -0,0 +1,207 @@
+---
+title: Getting Started with YAML
+permalink: /schema/getting-started/yaml
+category: Reference
+menuOrder: 2
+---
+
+<WarningBox heading={`🐣  Preview`}>
+
+YAML modelling is currently in Preview, and the syntax may change in a future
+version.
+
+</WarningBox>
+
+## Ensure deployment is using latest Cube
+
+First, ensure you're using Cube version `0.31.0` or higher. Then create a new
+`.yml` file in the `schema/` folder.
+
+## Syntax
+
+<InfoBox>
+
+1. All schema properties can be written in either camel-case or snake-case i.e.
+   `countDistinctApprox` can be written as `count_distinct_approx`,
+   `preAggregations` can be written as `pre_aggregations` etc.
+
+2. When referencing either `CUBE` or another foreign cube, ensure the reference
+   is wrapped in `{}` and the entire statement is wrapped in quotes i.e.
+   `'{CUBE}.user_id = {Users.id}'`.
```

Comment:
Let's use double quotes here. Also let's say that for strings you don't normally need to use quotes, unless you want your numbers to be treated as strings or you use special numbers, including `{}` for interpolation.